tools/rosgraph/test/isolated/test_roslogging_user_logger.py

```diff
@@ -0,0 +1,133 @@
+# Software License Agreement (BSD License)
```

Comment:
This file shouldn't repeat all the tests from the other file but only check the `UserCustomLogger`.